### PR TITLE
slim v3.5 (new formula)

### PR DESCRIPTION
**Pull request by terhorst**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

SLiM is an evolutionary simulation framework that combines a powerful engine for population genetic simulations with the capability of modeling arbitrarily complex evolutionary scenarios.

### CODE REVIEW

File: Formula/slim.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  desc "Forward simulator for population genetics and evolutionary biology"
+  homepage "https://messerlab.org/slim/"
+  url "https://github.com/MesserLab/SLiM/archive/v3.5.tar.gz"
+  sha256 "7f6f9b33416d0f5a6ab4e004f8f2cb5251b57ae270da7f4b3054b2135765a376"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV.cxx11
```
Comment (sjackman) [inline Formula/slim.rb:12]: Is this line needed?

Comment (sjackman) [reply Formula/slim.rb:12]: If it builds fine without `ENV.cxx11`, you can leave it out.

File: Formula/slim.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    mkdir "SLiM_build"
+    cd "SLiM_build"
+    system "cmake", "-DCMAKE_BUILD_TYPE=Release", "..", *std_cmake_args
+    system "make"
+    system "make", "install"
```
Comment (sjackman) [inline Formula/slim.rb:17]:
```suggestion
    mkdir "SLiM_build" do
      system "cmake", "-DCMAKE_BUILD_TYPE=Release", "..", *std_cmake_args
      system "make"
      system "make", "install"
    end
```

**Review by sjackman — CHANGES_REQUESTED** [review]
Use `mkdir … do`

File: Formula/slim.rb (hunk 1 above)
Comment (terhorst) [reply Formula/slim.rb:12]: It works without it.

**Comment by sjackman** [comment]
Thank you for your second contribution to Brewsci/bio, Jonathan! It's great to see repeat contributions. 🏆

**Pull request merged**